runtime/unwind/src/main/java/org/qbicc/runtime/unwind/Unwind.java

```diff
@@ -90,6 +90,8 @@ private Unwind() {
          * The function which frees this object.
          */
         public function_ptr<_Unwind_Exception_Cleanup_Fn> exception_cleanup;
+        public uint64_t private_1;
```

Comment:
It looks like the actual type of these fields should be `unsigned_long`.  It could be incorrect on 32-bit systems otherwise (`long` is often 32 bits on such systems).